spec/queries/site_query_spec.rb

```diff
@@ -8,42 +8,43 @@
   describe "#order_by" do
     subject(:result) { query.order_by(params) }
 
-    context "when sort[url]=asc" do
-      let(:request) { { sort: { url: :asc } } }
-
-      # Create audits with domains that will sort alphabetically
-      let!(:apple)  { create(:audit, :passed, :current, url: "https://apple.com") }
-      let!(:banana) { create(:audit, :passed, :current, url: "https://banana.org") }
+    context "when ordering by url" do
+      let!(:apple) { create(:audit, :passed, :current, url: "https://apple.com") }
+      let!(:app) { create(:audit, :passed, :current, url: "http://www.app.apple.com") }
+      let!(:banana) { create(:audit, :passed, :current, url: "https://www.banana.org") }
+      let!(:blog) { create(:audit, :passed, :current, url: "http://blog.beta.com") }
       let!(:carrot) { create(:audit, :passed, :current, url: "https://carrot.bio") }
-      let(:expected_ids) { [apple.site_id, banana.site_id, carrot.site_id] }
```

Comment:
Pas besoin d'en créer 5, il suffit d'en créer 2 ou 3 mais pas dans l'ordre alphabétique.
```suggestion
let!(:carrot) { create(:audit, :passed, :current, url: "https://carrot.bio") }
let!(:apple) { create(:audit, :passed, :current, url: "https://apple.com") }
let!(:banana) { create(:audit, :passed, :current, url: "https://www.banana.org") }
```